### PR TITLE
Add Hiking eBee image and backstory 

### PR DESCRIPTION
**Pull request by jgcolvin-iso**
Add Hiking eBee

### CODE REVIEW

**Review by qmonnet — CHANGES_REQUESTED** [review]
Thanks a lot!

The backstory looks good, with a few nits below. I've got some additional questions:

- What's the plan with regard to https://github.com/isovalent/ebeedex/issues/70? I don't suppose we plan to merge two versions of the hiking bee, so I take it you prefer to go with the one without the background?

- The picture itself is more stretched horizontally in comparison with the one I had (which admittedly looks a bit compressed, somehow it feels like we could have something in between - I've no idea which was the initial shape). Is this intended/OK?

- Also about the picture: Could you please crop to contents? I haven't checked the size of the picture. Please take a look at https://github.com/isovalent/ebeedex/blob/main/image-formatting.md (I'm happy to help if necessary).

File: README.md [hunk 1]
```diff
@@ -248,6 +248,14 @@ This eBeeDex lists all known eBees and their back story. If you find any more in
...
    </td>
   </tr>
+    <td> <img src="images/hiking-bee.png" alt="Hiking">
+  </td>
+   <td><a href="#Hiking eBee" id="Hiking eBee">#28: Hiking eBee</a>
+     <br />
+     <br />
+       Armed with a backpack filled with snacks and a sense of adventure, the Hiking eBee is always on the lookout for peak experiences. This eBee is no stranger to challenges. They carefully navigate rocky paths, leap across babbling streams, and scale steep cliffs, always driven by an unwavering appreciation for the outdoors. Along the way, they encounter fellow creatures of the mountain, from chirping birds to inquisitive squirrels, forging connections and cherishing the natural world. As they navigate challenging paths and form bonds with the mountain’s residents, they find kindred spirits among their fellow eBees, like the tech-savvy Electrician eBee, who illuminates their night hikes with digital lanterns.
```
[the remark this answers is not in the document]
Comment (jgcolvin-iso) [reply README.md:256]: *Electric lantern, not digital :)

[the remark this answers is not in the document]
Comment (jgcolvin-iso) [reply README.md:256]: Wouldn't want to scare any of the hive. Can update to: from deers grazing

[the remark this answers is not in the document]
Comment (qmonnet) [reply README.md]: Electric lanterns sound good to me :)

[the remark this answers is not in the document]
Comment (qmonnet) [reply README.md]: Sounds perfect

**Comment by jgcolvin-iso** [comment]
Thanks - some comments below! 

> * What's the plan with regard to [eBee spotted: Hiking eBee #70](https://github.com/isovalent/ebeedex/issues/70)? I don't suppose we plan to merge two versions of the hiking bee, so I take it you prefer to go with the one without the background?
I prefer the background one, I wasn't sure with the formatting rules whether the background was okay. If it is, lets use the background one. I have a cleaned up one attached here that removes the top left blue semi-circle. 
offsite-main-graphic-tr

> * The picture itself is more stretched horizontally in comparison with the one I had (which admittedly looks a bit compressed, somehow it feels like we could have something in between - I've no idea which was the initial shape). Is this intended/OK?
Not intended, I applied 577 x 577 pixels and the image transformed from the original but I did not run it through Gimp before for margins.

> * Also about the picture: Could you please crop to contents? I haven't checked the size of the picture. Please take a look at https://github.com/isovalent/ebeedex/blob/main/image-formatting.md (I'm happy to help if necessary).
I can run through this again with the one with the nice fall background, or if you want to feel free to update as well.

**Review by qmonnet — APPROVED** [review]
OK I cropped and resized.
Looks all good, thanks!

**Pull request merged**